framework_lib/src/commandline/clap_std.rs

```diff
@@ -145,6 +146,13 @@ struct ClapCli {
     #[arg(long)]
     kblight: Option<Option<u8>>,
 
+    /// Set the color of <key> to <RGB>. Multiple colors for adjacent keys can be set at once.
+    /// <key> <RGB> [<RGB> ...]
+    /// Example: 0 0xFF000 0x00FF00 0x0000FF
+    #[clap(num_args = 2..EC_RGBKBD_MAX_KEY_COUNT)]
```

Comment:
why is this 2?
You should allow EC_RGBKBD_MAX_KEY_COUNT to be bigger than 64. This number is orthogonal to how big a host command can be.

Reply:
it means at least two. offset and the first color

Reply:
>  You should allow EC_RGBKBD_MAX_KEY_COUNT to be bigger than 64. This number is orthogonal to how big a host command can be.

True, i can make it send multiple host commands